libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/RouteArrowUtils.kt

```diff
@@ -55,14 +55,16 @@ internal object RouteArrowUtils {
     }
 
     fun initializeLayers(style: Style, options: RouteArrowOptions) {
-        if (
-            !style.fullyLoaded ||
-            layersAreInitialized(style) ||
-            !style.styleLayerExists(options.aboveLayerId)
-        ) {
+        if (!style.fullyLoaded || layersAreInitialized(style)) {
             return
         }
 
+        val aboveLayerIdToUse = if (style.styleLayerExists(options.aboveLayerId)) {
+            options.aboveLayerId
+        } else {
+            null
+        }
```

Comment:
Would you be able to add a KDoc entry to the arrow options builder about `PRIMARY_ROUTE_TRAFFIC_LAYER_ID` constant and that it is the suggested variable to use for the optimized experience?